README.md

```diff
@@ -159,7 +159,7 @@ The relevant code for this sample is in the following files:
 
 ## Troubleshooting
 
-If you are repeatedly asked to Grant permissions. See the note in [SyncController](https://github.com/Azure-Samples/active-directory-dotnet-daemon-v2/blob/master/UserSync/Controllers/SyncController.cs) on how to clear your token cache.
+If you are repeatedly asked to Grant permissions, Sign-out and sign back in again to force a token cache refresh.
```

Comment:
Why is it so, @kalyankrishna1. It seems that there is something wrong?

Reply:
This was the bug fix. Removed this text